### DISCUSSION TITLE
Extending Blog Posts. Articles & Magazines?

### DISCUSSION THREAD

**Discussion by gribbles67** [funmusicplace/mirlo#530] [Ideas]
Mirlo already has the ability for an artist to create blog posts and embed individual tracks in the post. On the face of it it’s been implemented to let musicians blog about their releases and to get general news out to their fans/followers on Mirlo.

On the back of some discussion on discord about how the Bandcamp daily articles were pretty useful. I started thinking that it would be really interesting if Mirlo aimed for a similar feature, but  rather than employing a bunch of gatekeepers to write them, allowing the articles to come from the community

Initially, this was just thinking about reviewing blog posts that folk were writing, and promoting the interesting ones to the homepage, or wherever – which is still a form of gatekeeping, I guess!

But then I thought that it would be good to allow individual users on Mirlo to create their own  “Mirlo Dalies“ in a nutshell, using the blog post feature to write articles and have them presented on their own page. 

On reflection I think that there’s a strong analogy between a blog post as an article for a writer and an individual track for a musician. In a similar way that individual tracks can be compiled into a playlist or album, then individual blog posts or articles can be compiled into a magazine or newsletter (a playlist of articles…)

This would allow writers to create newsletter/magazines and present them on the page. Like an album/playlist cover, you could allow a magazine cover to be uploaded. I think initially allowing a couple of fixed presentations for a magazine around a table of contents linking to the articles, with maybe the ability to highlight featured articles would be fine.

You could tie them into subscriptions in a really meaningful way to allow the reading of free articles, and then subscribe to get more in depth or other features.

That gets really interesting when you consider that common functionality on other platforms allows you to compile tracks from different artists into a playlist. If you then apply that to blog post/articles, then you’re giving people the ability to compile their own magazines out of articles other people have written…

Add some likes in there, and you’ve got the beginnings of a self curating “Mirlo Daily“ of the most popular articles across the platform in a day/week/month or whatever.

Again, analogous to individual tracks, if you add in tagging you can start to look at doing similar things with articles and writing, as is being discussed for music in the [other topic on tagging](https://github.com/funmusicplace/mirlo/discussions/166)  – i.e. looking at all the articles written about live hip-hop events in the Chicago area. [tags: live, hip-hop, Chicago)

It may be worthwhile making the distinction between users being musicians and writers, but that might just be adding extra complexity for little gain.

If you allow the embedding of articles out of Mirlo into other platforms, then Artists will probably add those to their websites, and you’ve got an automatic link back to Mirlo.

Mirlo then becomes a way of building communities around music by folk other than the musicians that use it, as opposed to ‘just’ a storefront for musicians. I think this ties in with the discussion about how to differentiate from other platforms and hopefully short-circuit the cynicism that potential users may have around “just another platform to sell, upload my music to". 

As I was writing this, it’s just occurred to me that labels could probably make good use of a “magazine“ feature as well.

I think that’ll do to get the ball rolling, but there is a little voice in my head saying there’s probably more to this! 

.g

**Comment by gribbles67** [comment]
Of course, any articles that embed a track/album/playlist/artist profile could be listed on the relevant page, so artists can see who is writing about them and listeners can have a read about the music they have discovered. 

Articles automatically added to a page in this way should be hidden by default and then the artist can choose to display them. 

This brings to mind another metric that could be used - album/track most written about…

**Reply by simonv3** [reply]
I like this!

**Comment by simonv3** [comment]
Going to add this as a screenshot of a substack "magazine" page 
![afbeelding](https://github.com/funmusicplace/mirlo/assets/485583/a07c4f15-b8d9-409e-bb25-733d4fd6c851)

**Comment by Cwpute** [comment]
> It may be worthwhile making the distinction between users being musicians and writers, but that might just be adding extra complexity for little gain.

Oh no, not useless at all. A big, big big part of the music community is about discoverability, and catering for people who produce that discoverability is the best way to have them embrace your platform and further nurture the community !
As an example, a cousin of mine is making electronic music and DJing, and the best way for him to have his music be discovered is if someone with lots of followers make a mix or a DJ-set including your music. Because of this, his music MUST be on Soundcloud and LastFM and Mixcloud, *because the people who curate his style of music are*.

I don't think you need to have curators (i prefer that over "writers", much broader) have a specific account model.
Instead, like artists do in Mirlo, you could allow any user to post articles on their user page... but to prevent them from posting only text articles, only allow them to do so if they include at least 2 tracks from Mirlo.space in it.
Another way to do it would be to allow users to make playlists from tracks on Mirlo, and accompany these with text. Effectively it's the same as above, but you put the playlist first.

**Comment by Cwpute** [comment]
We might want to look atwhat Rauversion themselves did with their articles: https://rauversion.com/
